android/build.gradle

```diff
@@ -73,10 +76,12 @@ repositories {
 dependencies {
     //noinspection GradleDynamicVersion
     implementation 'com.facebook.react:react-native:+'  // From node_modules
+    implementation("com.squareup.okhttp3:okhttp:4.4.0") // necessary for the cookie setting to work
```

Comment:
React Native core use okhttp 3.12.1, there is no collision to use this version (4.4.0) ?

Reply:
From the team

> ...might cause breakage if API changed
> I don’t expect FB to upgrade Okhttp because it dropped Android 4.x support

We should find an alternative.

Reply:
Fair point guys. In such case we have 2 options: 

A. I can match the same version as RN uses. That shouldn't be a problem. 

B. Find a different option for the cookie string make up. 

I'll start with A and look at options for B.

Reply:
Got rid of okhttp as requested. I tested more thoroughly and indeed it would cause issues.